api/src/main/java/org/pmiops/workbench/rdr/RdrMapper.java

```diff
@@ -0,0 +1,118 @@
+
+  default OffsetDateTime toModelOffsetTime(Timestamp dbTime) {
+    return dbTime.toLocalDateTime().atOffset(offset);
+  }
+
+  default RdrWorkspace.StatusEnum toModelStatus(WorkspaceActiveStatus workspaceActiveStatus) {
+    return workspaceActiveStatus == WorkspaceActiveStatus.ACTIVE
+        ? RdrWorkspace.StatusEnum.ACTIVE
+        : RdrWorkspace.StatusEnum.INACTIVE;
+  }
+
+  default boolean toModelFocusOnUnderrepresentedPopulation(
+      Set<SpecificPopulationEnum> dbSpecificPopulationSet) {
+    return dbSpecificPopulationSet != null && dbSpecificPopulationSet.size() > 0;
+  }
+
+  default RdrWorkspaceDemographic toModelWorkspaceDemographic(
+      Set<SpecificPopulationEnum> dbPopulationEnumSet) {
+    RdrWorkspaceDemographic rdrDemographic = new RdrWorkspaceDemographic();
+
+    rdrDemographic.setAccessToCare(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.ACCESS_TO_CARE)
+            ? RdrWorkspaceDemographic.AccessToCareEnum.NOT_EASILY_ACCESS_CARE
+            : RdrWorkspaceDemographic.AccessToCareEnum.UNSET);
+
+    rdrDemographic.setDisabilityStatus(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.DISABILITY_STATUS)
+            ? RdrWorkspaceDemographic.DisabilityStatusEnum.DISABILITY
+            : RdrWorkspaceDemographic.DisabilityStatusEnum.UNSET);
+
+    rdrDemographic.setEducationLevel(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.EDUCATION_LEVEL)
+            ? RdrWorkspaceDemographic.EducationLevelEnum.LESS_THAN_HIGH_SCHOOL
+            : RdrWorkspaceDemographic.EducationLevelEnum.UNSET);
+
+    rdrDemographic.setIncomeLevel(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.INCOME_LEVEL)
+            ? RdrWorkspaceDemographic.IncomeLevelEnum.BELOW_FEDERAL_POVERTY_LEVEL_200_PERCENT
+            : RdrWorkspaceDemographic.IncomeLevelEnum.UNSET);
+
+    rdrDemographic.setGeography(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.GEOGRAPHY)
+            ? RdrWorkspaceDemographic.GeographyEnum.RURAL
+            : RdrWorkspaceDemographic.GeographyEnum.UNSET);
+
+    rdrDemographic.setSexualOrientation(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.SEXUAL_ORIENTATION)
+            ? RdrWorkspaceDemographic.SexualOrientationEnum.OTHER_THAN_STRAIGHT
+            : RdrWorkspaceDemographic.SexualOrientationEnum.UNSET);
+
+    rdrDemographic.setGenderIdentity(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.GENDER_IDENTITY)
+            ? RdrWorkspaceDemographic.GenderIdentityEnum.OTHER_THAN_MAN_WOMAN
+            : RdrWorkspaceDemographic.GenderIdentityEnum.UNSET);
+
+    rdrDemographic.setSexAtBirth(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.SEX)
+            ? RdrWorkspaceDemographic.SexAtBirthEnum.INTERSEX
+            : RdrWorkspaceDemographic.SexAtBirthEnum.UNSET);
+
+    rdrDemographic.setRaceEthnicity(
+        dbPopulationEnumSet.stream()
+            .map(RdrExportEnums::specificPopulationToRaceEthnicity)
```

Comment:
You may want to find out when this returns null. Maybe a unit test there. In general, if you have to do a filter on non-null, there might be a constraint you could add upstream so this doesn't happen. Not always though.